### ISSUE TITLE
can you have a feature to change the input video to another one?

### ISSUE THREAD

**Issue by zumzum360** [stephenswetonic/ytpai#8]
well i was using your website, so? i have to reload the site to change the video... so? can you do a feature to change the input video to change to other videos?

**Comment by stephenswetonic** [comment]
Yes that is something I've wanted to add. Then the extracted words could be added all on one page.

**Comment by zumzum360** [comment]
okay then

**Comment by stephenswetonic** [comment]
I added a close button for now, but handling multiple source will take some more work. When it's done i'll close this issue.
